Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:slim-buster
```

Comment:
can we use alpine?

Reply:
Sure, but do note that it won't have arm v7/32-bit support.
https://hub.docker.com/layers/library/rust/alpine3.17/images/sha256-3dd0bb6f134635fe40dd9c18bd9603f9d90ce3538ac25ae3e69b9b127137acf2?context=explore

https://hub.docker.com/layers/library/rust/slim-buster/images/sha256-ae6ae30b64a325291c8623c9dcf383210a32b5d4b89a4923fffdd00501785afd?context=explore

This would mean that 32-bit rpi's can't run this container. Also the alpine one is 20MB bigger strangely enough.

Reply:
Let me know if you prefer to use alpine still, if not, i can resolve this convo and we can discuss any other points if there's any :)